tutorials/AddingNewModel.md

```diff
@@ -0,0 +1,28 @@
+<h1>Adding New Model - Caffe</h1>
+
+1. For Setup Instructions. - https://github.com/Cloud-CV/Fabrik/blob/master/README.md
+2. Open the <b>Example folder</b>.
+3. Open the <b>Caffe folder</b> and add the new model in the form of .prototxt file.
+4. Then add your entry to the frontend by adding the following line to <b>topbar.js</b>, in this the id should be the name of your prototxt without the extension.
```

Comment:
– *front-end
– add a link to topbar.js
– use code highlighting on `id`, and reference the example provided below